### DISCUSSION TITLE
Add characters to players - remove spectator warning

### DISCUSSION THREAD

**Discussion by Handydan** [Jagusti/fvtt-wfrp4e-gmtoolkit#252] [General]
Sorry for a basic question. How do one add a character to a player so that the Spectator-warning dissapears? Is it sufficient with right click on Actor Image -> Configure Ownership?

**Comment by Jagusti** [comment]
No need to apoologise  - this is an oft-asked thing. If I had an FAQ, I would probably I'd it there #notetoself  ...

1. Make sure they have **ownership** of the character you want them to take: right-click the character in the actor directory and configure ownership).
2. Then you need to **assign** a character to the player: right-click their name in the player list at the bottom left and tehre is an option that's something like assign

**Comment by Jagusti** [comment]
You can also suppress the spectator warning from showing at all, in case, eg, you have fake players that you use for streaming. Check the User Settings as per the wiki: https://github.com/Jagusti/fvtt-wfrp4e-gmtoolkit/wiki/spectators.
